tests/README.md

```diff
@@ -0,0 +1,25 @@
+Install the dependencies using
+
+`pip install -r requirements/CI/requirements.txt`
```

Comment:
if you're saying `python3` below don't you need to also say `pip3`?

Reply:
Ah, possibly. No harm in that. Will change.